src/GitHub.VisualStudio/Services/UsageTracker.cs

```diff
@@ -11,21 +11,21 @@
 using GitHub.Extensions;
 using System.Threading.Tasks;
 using GitHub.Helpers;
+using System.Threading;
 
 namespace GitHub.Services
 {
-    public class UsageTracker : IUsageTracker
+    public sealed class UsageTracker : IUsageTracker, IDisposable
```

Comment:
LOL! I do this all the time in apps. Less so in libraries. But it's very convenient to just seal them until you need them not to be sealed. 😄